haskell/experimental/defs.bzl

```diff
@@ -7,6 +7,7 @@ load(
 )
 load("//haskell:private/cc_libraries.bzl", "haskell_cc_libraries_aspect")
 
+# TODO[GL] should we have repl_ghci_args here?
```

Comment:
Not sure about this one.

Reply:
Probably not for now. Seems like something we can try to add later if a user wants it.